src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Services/ProviderEmailServiceTests/WhenISendEmailToAllProviderRecipients.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerCommitments.Application.Services;
+using SFA.DAS.EmployerCommitments.Domain.Configuration;
+using SFA.DAS.EmployerCommitments.Domain.Interfaces;
+using SFA.DAS.EmployerCommitments.Domain.Models.Notification;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.Notifications.Api.Client;
+using SFA.DAS.Notifications.Api.Types;
+
+namespace SFA.DAS.EmployerCommitments.Application.UnitTests.Services.ProviderEmailServiceTests
+{
+    [TestFixture]
+    public class WhenISendEmailToAllProviderRecipients
+    {
+        private ProviderEmailService _providerEmailService;
+        private Mock<IProviderEmailLookupService> _providerEmailLookupService;
+        private Mock<INotificationsApi> _notificationsApi;
+
+        private long _providerId;
+        private string _providerLastUpdateEmailAddress;
+        private CommitmentNotificationConfiguration _commitmentNotificationConfiguration;
+        private List<string> _exampleRecipients;
+        private EmailMessage _exampleValidEmail;
+
+        private List<Email> _sentEmails;
+
+        private Func<Task> _act;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _providerId = 1;
+            _providerLastUpdateEmailAddress = "AlternativeEmailAddress";
+
+            _commitmentNotificationConfiguration = new CommitmentNotificationConfiguration {SendEmail = true};
+
+            _exampleRecipients = new List<string> { "Recipient1", "Recipient2", "Recipient3" };
```

Comment:
It might be worth supplying valid email addresses, in case the CUT ever introduces email verification, or fails with valid email addresses.

I tend to use @example.com, as it is reserved (https://www.iana.org/domains/reserved) and safe if real emails are sent accidentally.